packages/@react-spectrum/button/test/ClearButton.test.js

```diff
@@ -29,7 +28,6 @@ describe('ClearButton', function () {
   it.each`
     Name                | Component      | props
     ${'v3 ClearButton'} | ${ClearButton} | ${{onPress: onPressSpy}}
```

Comment:
Should we also remove the v3 references that existed too distinguish from v2 since v2 is gone?

Reply:
Next level clean up